app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java

```diff
@@ -38,6 +41,16 @@ public class OutfitCreationActivity extends AppCompatActivity {
     FirebaseAuth mAuth;
     Context context;
 
+    //clothing IDs
+    String topID = "";
+    String bottomsID = "";
+    String shoesID = "";
+    String headwearID = "";
+    String socksID = "";
+    String otherID = "";
+    String outfitName = "";
+    String outfit[] = new String[7];    //[0]: name [1]: top [2]: bottoms [3]: shoes [4]: headwear [5]: socks [6]: other
```

Comment:
Please make sure you don't comment out anything important so the files don't have errors.

Reply:
got it